### ISSUE TITLE
GIFT phase II Guatemala: uploading dataset and using OS spending tools

### ISSUE THREAD

**Issue by DianaMPG** [openspending/openspending#1167]
This relates to #1165

**Comment by DianaMPG** [comment]
Change request 

The filters should portray the following hierarchies: 


Ministry (admin unit)
Sub-units
Programm 
Activitiy 

Diana will check if this is already the case with the current arrangement of filters. Team Guatemala will check on the current filter arrangement  and gives feedback
--- 
Change request

Some of the columns might need rearranging, Team Guatemala gives us feedback on this

3. OS Viewer enhancements 

Visualisation types

Pie Charts: 
too many columns make the pie chart obsolete as too understanding the information correctly; readability etc. 

On Github there is already an issue that addresses this: https://github.com/openspending/openspending/issues/1197

The following recommendations can be found in the report:
Pie Chart
Limitation of 5 categories per chart
A chart with too many values is difficult to read and understand. It decreases the understanding and impact of the visualization.
In order to avoid these traps, we can either display a warning message to the final user if she has more than 5 categories to display or group smaller values into an “other” or “miscellaneous” category.
Limit the range of colors used
Following aesthetic minimalism guidelines can convey the message more efficiently.
Edward Tufte suggests to remove all elements that are not related to data and actively interfere with the reading and understanding of a visualisation (Tufte, 1983).
Extension: The donut chart visualisation
Donut charts are interesting as they allow the reader to directly see the absolute values of each slice and of the total.

If this suits Team Guatemala as or has any other suggestion to the issue, we will add a comment to this issue,  which we will be working on over the cause of 2017. 

4. Pipeline session	

Since Guatemala has expenditure/spending data that needs daily update, they would be very much be interested in attending a pipeline session, maybe together with Uruguay to make the translation part easier. /

**Comment by akariv** [comment]
@DianaMPG do we have an update on these items?

**Comment by DianaMPG** [comment]
@akariv this issue merges with #1166

**Comment by akariv** [comment]
So can we please separate into individual tasks? It's hard to follow up on multiple, different issues when they're all bundled like this.

**Comment by akariv** [comment]
@DianaMPG

**Comment by DianaMPG** [comment]
@akariv done
